code/modules/clothing/head/helmet.dm

```diff
@@ -31,36 +31,47 @@
 	name = "warden's hat"
 	desc = "It's a special helmet issued to the Warden of a securiy force. Protects the head from impacts."
 	icon_state = "policehelm"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
 	flags_inv = 0
 
 /obj/item/clothing/head/helmet/warden/commissar
 	name = "commissar's cap"
 	desc = "A security commissar's cap."
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
 	icon_state = "commissarcap"
 
 /obj/item/clothing/head/helmet/HoS
 	name = "head of security hat"
 	desc = "The hat of the Head of Security. For showing the officers who's in charge."
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
 	icon_state = "hoscap"
-	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
-	flags_inv = HIDEEARS
 
 /obj/item/clothing/head/helmet/HoS/dermal
 	name = "dermal armour patch"
 	desc = "You're not quite sure how you manage to take it on and off, but it implants nicely in your head."
 	icon_state = "dermal"
+	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	allow_hair_covering = FALSE
 
+/obj/item/clothing/head/helmet/HoS/tacticalhelm
+	name = "commander tactical helmet"
+	desc = "A special Internal Security Division helmet designed to protect the precious craniums of important installation security officers."
+	icon_state = "hos_helmet"
+	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
+	flags_inv = HIDEEARS
+
 /obj/item/clothing/head/helmet/hop
 	name = "crew resource's hat"
-	desc = "A stylish hat that both protects you from enraged former-crewmembers and gives you a false sense of authority."
+	desc = "A stylish hat that gives you a false sense of authority."
 	icon_state = "hopcap"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
 	flags_inv = 0
 
 /obj/item/clothing/head/helmet/formalcaptain

```

Comment:
This should be repathed to be a normal hat, not a child of helmet, since the armor is removed.